src/m_menu.c

```diff
@@ -148,10 +148,12 @@ background_t menu_background;
         ((condition) ? (item.m_flags |= S_DISABLE) : (item.m_flags &= ~S_DISABLE))
 
 #define MI_PREV(page) \
-  {"<- PREV", S_SKIP|S_PREV, m_null, M_X_PREV, M_Y_PREVNEXT, {page}}
+  {"<- PREV", S_SKIP|S_PREV, m_null, M_X_PREV, M_Y_PREVNEXT, {page}, \
```

Comment:
Do reset buttons need s similar fix?

Reply:
No, it's `mrect_t` depends on the patch size (there are PWADs with reset buttons, for example Dimension of the Boomed).